### PR TITLE
Add reasoner to rosinstall

### PR DESCRIPTION
**Pull request by marioney**
Adds mros_reasoner to [.rosintall](https://github.com/rosin-project/metacontrol_sim/blob/master/metacontrol_sim.rosinstall) file and modifies parameters on [metacontrol_sim_reasoner.launch](https://github.com/rosin-project/metacontrol_sim/blob/master/launch/metacontrol_sim_reasoner.launch)

### CODE REVIEW

**Review by chcorbato — APPROVED** [review]
Excellent!

File: metacontrol_sim.rosinstall [hunk 1]
```diff
@@ -24,3 +24,18 @@
...
+    local-name: metacontrol_msgs
+    uri: https://github.com/rosin-project/metacontrol_msgs
+- git:
+    local-name: mc_mdl_tomasys
+    uri: https://github.com/tud-cor/mc_mdl_tomasys.git
+    version: v0.1
+- git:
+    local-name: catkin_virtualenv
+    uri: https://github.com/locusrobotics/catkin_virtualenv.git
+    version: 0.6.1
```
Comment (gavanderhoorn) [inline metacontrol_sim.rosinstall:41]: Not sure where this was ticketed again, but `catkin_virtualenv` was synced to Melodic: [status_page/ros_melodic_default.html?q=catkin_v](http://repositories.ros.org/status_page/ros_melodic_default.html?q=catkin_v).

Comment (marioney) [reply metacontrol_sim.rosinstall:41]: > Not sure where this was ticketed again

It's on this issue  https://github.com/tud-cor/mc_mros_reasoner/issues/88

I'll test it, and if it works and change it

Comment (marioney) [reply metacontrol_sim.rosinstall:41]: OK, I've removed it and also fixed the mros_reasoner version in https://github.com/rosin-project/metacontrol_sim/commit/29f30f76be693aa648d858ae5e6be37d1bc69990

**Pull request merged**